lib/ansible/galaxy/collection.py

```diff
@@ -82,7 +81,10 @@ def __init__(self, namespace, name, b_path, source, versions, requirement, force
         self.add_requirement(parent, requirement)
 
     def __str__(self):
-        return to_text("%s.%s" % (self.namespace, self.name))
+        return to_native("%s.%s" % (self.namespace, self.name))
+
+    def __unicode__(self):
+        return u"%s.%s" % (self.namespace, self.name)
```

Comment:
maybe
```suggestion
        return to_text(str(self))
```

cc @abadger?